repo/packages/J/jenkins/29/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "3.0",
```

Comment:
Any reason this is not `"4.0"`?

Reply:
version 4.0 adds the upgradesFrom and downgradesTo fields in the package definition, currently we have any supported jenkins upgrade strategy (to my knowledge at least), so the package does not use these fields.